.travis.yml

```diff
@@ -1,36 +0,0 @@
-dist: xenial
-env:
-  global:
-  - secure: |-
-      IzTN2uuR4nsJ8+1IRmLXxn29+MK8PuKYgArbAU/aQPPm1vIUdMv0AYF/rq9N
-      YBdJJqfUHwOcGBFuwX1oKp4ykb6VTdYGOOCAN0j3IWtDXMkFM+qSTsLishyw
-      wSR11Z5SzTPxMGrY8cOupSH8mckmJvecomcYLctZdYwiP72+rSs=
-addons:
-  apt:
-    update: true
-    packages:
-      - texlive-latex-base
-      - texlive-latex-extra
-      - texlive-fonts-extra
-      - texlive-fonts-recommended
-      - texlive-lang-english
-      - texlive-lang-italian
-      - texlive-xetex
-      - latexmk
-      - python3-sphinx
-
-# Travis runs ./gradlew assemble and ./gradlew check
-# which, according to build.gradle, will run all
-# jar, test and documentation tasks.
-# See: 
-#  - build.gradle
-#  - https://docs.travis-ci.com/user/languages/java/
-language: java 
-jdk:
-  - openjdk8
-  - openjdk9
-  - openjdk10
-  - openjdk11
-after_success: 
-  - bash <(curl -s https://codecov.io/bash)
```

Comment:
Can you please add to Azure Pipelines the upload of code coverage results to codecov?

Reply:
I've added 
 - the [task to publish the test results](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/test/publish-test-results?view=azure-devops&tabs=yaml), which feeds the JUnit XML results files to DevOps - you can see them in the test result tab of [this build](https://dev.azure.com/pviotti/gh-pipelines/_build/results?buildId=15&view=ms.vss-test-web.build-test-results-tab)
- the script to update codecov. It still requires the codecov token, which should be set through the web interface - see [here](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#secret-variables)